contents/docs/libraries/next-js/index.mdx

```diff
@@ -153,9 +153,9 @@ export function PHProvider({
 
 The problem with this is that it can cause a [hydration and/or mismatch error](https://nextjs.org/docs/messages/react-hydration-error) like `Warning: Prop dangerouslySetInnerHTML did not match.`.
 
-### Why does the pageview component need a `useEffect`? 
+### Why did the pageview component need a `useEffect`? 
 
-Using a `useEffect` hook is the simplest way to accurately capture pageviews. Other approaches include:
+Before changing to using `history_change` to capture pageviews, we suggested using a `useEffect` hook to capture pageviews. This is because it's the simplest way to accurately capture pageviews. Other approaches include:
```

Comment:
We might need a different callout here, specifying we *used* to suggest `useEffect` for this

Reply:
This sentence does specify we used to suggest useEffect I think